Majasdarbs/04-majasdarbs/Gvido-CV-responsive/css/styles.css

```diff
@@ -0,0 +1,201 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: rgb(0, 0, 0);
+    color: white;
+    font-family: 'Roboto', 'Arial', 'sans-serif'; 
+    font-size: 16px;
+}
+
+/* Navigation bar >>>>>>>>>>>>>>>>>>>>>*/
+.nav-bar {
+    display: flex;
+    justify-content: center;
+    background-color: white;
+    font-size: 16px;
+    position: fixed;
+    width: 100vw;
+    top: 0;
+}
+
+.nav-bar a{
+    text-decoration: none;
+    color: black;
+    padding: 0.3rem;
+    margin-left: 10px;
+}
+
+.nav-bar a:hover {
+    color: rgb(255, 255, 255);
+    padding: 1rem;
+    background-color: rgb(0, 0, 0);
+    transition: 0.7s;
+}
+
+/* Header >>>>>>>>>>>>>>>>>*/
+
+/* ABOUT ME */
+#about-me {
+    margin-top: 2rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    background-image: url(../img/bg-pic.jpg);
+    background-size: cover;
+    text-align: center;
+}
+
+#about-me a{
+    text-decoration: none;
+    color: white; 
+    visibility: hidden;
+}
+
+#about-me:hover svg,
+#about-me:hover h1 
+{
+    visibility: hidden;
+}
+
+#about-me:hover a {
+    visibility: visible;
+    padding: 5px;
+    border-radius: 5px;
+    background-color: gray;
+}
+
+/* MAIN Text part >>>>>>>>>>>>>>>>>>>>>>>>> */
+.main-content {
+    padding-top: 50px;
+    padding-bottom: 50px;
+    margin: auto;
+    width: 70%;
+}
+
+.main-content h2 {
+    text-transform: uppercase;
+    padding-bottom: 1rem;
+}
+
+.main-content ul {
+    padding-left: 40px;
+}
+
+/* WORK EXPERIENCE */
+#work-experience {
+    padding-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border-bottom: 2px solid rgb(53, 53, 53);
+}
+
+#work-experience h3 {
+    padding-top: 1rem;
+}
+
+/* EDUCATION */
+#education {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border-bottom: 2px solid rgb(53, 53, 53);
+}
+
+/* SKILLS */
+#skills {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border-bottom: 2px solid rgb(53, 53, 53);
+}
+
+/* FAQ */
+#faq {
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    display: flex;
+    flex-direction: column;
```

Comment:
in future if you see, that you are repeating styles (for different sections in this case) consider adding class to the section elements and add generic styles, in you case:
    padding-top: 2rem;
    padding-bottom: 2rem;
    display: flex;
    flex-direction: column;
    align-items: flex-start;